.gitignore

```diff
@@ -1,52 +1,39 @@
 # dependencies
-node_modules
+node_modules/
 
 # coverage
 coverage
 .nyc_output
 
+.pytest_cache/
+
 # production
-build
+build/
+dist/
+*.egg-info
+
+# Environments
+venv/
+cellxgene/
+
+# client build
+server/app/web/static/css/
+server/app/web/static/img/
+server/app/web/static/js/
+server/app/web/templates/index\.html
+
+# Jupyter Notebook
+.ipynb_checkpoints
+*.ipynb
 
 # misc
 .DS_Store
 npm-debug.log
 .vscode
-
-data
```

Comment:
I still have a `data` folder - can I safely delete?

Reply:
Does it have EM2 files in it? If so yes. 

`data` was originally in there because that is where I was storing the various datasets built for EM2 in there.